src/main/resources/logback.xml

```diff
@@ -63,4 +63,7 @@
   <!-- Suppress an inconsequential IOException that is logged at debug level. -->
   <logger name="org.glassfish.grizzly" level="info" />
 
+  <!-- optional logging configuration to define your own custom logging needs -->
+  <include optional="true" resource="customLog.xml" />
```

Comment:
Other config files use hyphens (router-config.json,build-config.json).

Perhaps we should name it `custom-log.xml`? WDYT?

Reply:
I agree. There seems to be another standard  config file called logback-test.xml. So I renamed the file:

logback-addendum.txt